### PR TITLE
Download images

### LINKED ISSUE #91: Option to export to various types of files
PDF, image, etc.

### PR DESCRIPTION
**Pull request by Darokrithia**
Ready to roll

Fixes #91

### CODE REVIEW

**Comment by Darokrithia** [comment]
There was one conflict which was easy to resolve.  In master, the nav bar was called sive nav bar.  In download it is called side nav bar

File: site/public/js/controllers/SaveNavController.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// var SaveNavController = (function() {
```
Comment (LeonMontealegre) [inline site/public/js/controllers/SaveNavController.js]: What is this?
It seems like it does nothing so please remove

File: site/app/views/CircuitPageView.php [hunk 2]
```diff
@@ -151,6 +157,7 @@ public function getOutput($user, $config, $itemNavConfig) {
         }
 
         $return .= <<<HTML
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/jspdf/1.4.1/jspdf.debug.js" integrity="sha384-THVO/sM0mFD9h7dfSndI6TS0PgAGavwKvB5hAxRRvc0o9cPLohB0wb/PTA7LdUHs" crossorigin="anonymous"></script>
```
Comment (LeonMontealegre) [inline site/app/views/CircuitPageView.php]: Always put scripts from other sources in the <head> tag of the HTML doc

Comment (Darokrithia) [reply site/app/views/CircuitPageView.php]: done

Comment (LeonMontealegre) [reply site/app/views/CircuitPageView.php]: @Darokrithia 
Push the commit

**Comment by Darokrithia** [comment]
@LeonMontealegre fixed both issues and pushed.  Should be ready for merge

**Pull request merged**